frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.unit.spec.js

```diff
@@ -35,20 +35,8 @@ describe("SmartScalar", () => {
       ["2019-11-01T00:00:00", 120, 220],
     ];
     const insights = [
-      {
-        "last-value": 120,
-        "last-change": 0.2,
-        "previous-value": 100,
-        unit: "month",
-        col: "Count",
-      },
-      {
-        "last-value": 220,
-        "last-change": 0.1,
-        "previous-value": 200,
-        unit: "month",
-        col: "Sum",
-      },
```

Comment:
These insight values are now computed in the frontend automatically, so we remove them in these unit tests to ensure we’re using those instead.